.travis.yml

```diff
@@ -35,4 +35,5 @@ go:
   - 1.7.4
   - 1.7.5
   - 1.8
+  - 1.9
```

Comment:
Shouldn't we look to the future and add coverage for 2.0 also?

Reply:
Sadly Travis lacks the ability to support unreleased versions of go.

You may need to open an issue with them.